content/1_docs/2_cookbook/9_setup/0_kirby-in-a-nutshell/recipe.txt

```diff
@@ -0,0 +1,318 @@
+Title: Kirby in a nutshell
+
+----
+
+Published: 2019-10-03
+
+----
+
+Description: Or what you wish you had known when you started your first Kirby project.
+
+----
+
+Text:
+
+Everyone has their own way of going about things. What works for you, might not work for others. And the things to consider vary for each project, depending on whether you are migrating from another system or whether you are trying to set things up from scratch, whether your project is small or really huge, and so on.
+
+So this recipe is not so much about best practices, it is rather a recap of how Kirby works, what you might want to keep in mind when you start your project with Kirby, what questions you might want to ask yourself and about possible approaches, no matter if you are completely new to web development, or if you come from other systems.
+
+## It's all about your project
+
+Kirby is all about giving you the freedom and the tools to create what you want, the way you want–for yourself or your clients. The philosophy is to keep things as simple as possible and to add only what is needed. You can start with a very basic setup and then extend later when new requirements come up.
+
+However, this freedom to create also means that you basically start from scratch and then build your project step by step, a bit like building with Lego™ blocks. It means that you need to know the basics like HTML and CSS, and some PHP (or be prepared to learn what you need along the way). While quite a few themes for Kirby are available to get you jump-started on https://getkirby-themes.com, you will have to touch code to make changes if these themes are not a 100% fit. The idea behind Kirby is to empower you to create your own unique site and not make your project look like thousands of others out there.
```

Comment:
Is the getkirby-themes site not linked on purpose?

Reply:
It is?

Reply:
That's interesting. I didn't know that Markdown automatically converted text to links without even using angle brackets. 😮